pivotx/modules/module_parser.php

```diff
@@ -1036,6 +1036,61 @@ function renderFeed() {
 
     }
 
+
+    // Determine if the page is being offset. (Zero offset is the same as no offset).
+    function hasOffset()
```

Comment:
Left over of a refactor into a separate function 
(not linked to db corruption)